### DISCUSSION TITLE
How to install / update on Home Assistant

### DISCUSSION THREAD

**Discussion by Git2tony** [sbidy/wiz_light#76] [General]
Great that you're making this!

I installed it manually (following the steps from your github) a month or so back and recently tried an automation in HA, but get an error... and it's not clear what is the problem so I thought I should update the wiz-light integration.

Now I'm wondering, is there a way to add this repository to HA, either directly via addons or via HACS? (I'm a n00b, sorry...so please do ELIF).

I'm asking this, because the 'manual install' doesn't 'show' anywhere in HA (to my knowlegde) and I don't know how to update it now... might be using some sort of sudo command?

It would be really nice tho if I could actually see it via the integrations in HA so I could just update it as such?

Again, thnx for making this possible!

**Comment by sbidy** [comment]
I'm currently working on a integration into the HA core. Status: https://github.com/home-assistant/core/pull/44779

**Reply by Git2tony** [reply]
That is awesome news, and much appreciated! (will I, when it is ready, need to uninstall the current setup first?).
